apps/src/templates/projects/ProjectWidget.jsx

```diff
@@ -10,6 +13,46 @@ const ProjectWidget = React.createClass({
     projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired
   },
 
+  statics: {
+    setupProjectWidget() {
+      $.ajax({
+        method: 'GET',
+        url: `/v3/channels`,
+        dataType: 'json'
+      }).done(projectLists => {
+        const widget = document.getElementById('projects-widget');
+        projectLists.sort((a, b) => {
+          if (a.updatedAt < b.updatedAt) {
+            return 1;
+          } else {
+            return -1;
+          }
+        });
+        projectLists = projectLists.filter(project => {
+          return !project.hidden;
+        });
+        projectLists = projectLists.slice(0,4);
+        let convertedProjectList = [];
+        for (let i = 0; i < projectLists.length; i++){
+          let convertedProject = {
+            name: projectLists[i].name,
+            channel: projectLists[i].id,
+            thumbnailUrl: projectLists[i].thumbnailUrl,
+            type: projectLists[i].projectType,
+            updatedAt: projectLists[i].updatedAt
+          };
+          convertedProjectList.push(convertedProject);
```

Comment:
On my local machine I have one project in `projectLists` that has `null` for its `projectType`.  I can filter out to prevent errors in the console for that case, but is there additional filtering for validity I should be doing before this feature goes live?